### DISCUSSION TITLE
Missing Announcement Regarding Goerli Deprecation in zksync-developers README

### DISCUSSION THREAD

**Discussion by samui68400** [zkSync-Community-Hub/zksync-developers#334] [General] [answered]
### Environment

Testnet

### Provide a brief description of the functionality you're trying to implement.

I am seeking to address an omission in the README file of the zksync-developers GitHub repository regarding the announcement of the deprecation of the Goerli testnet and the introduction of the zkSync Sepolia Testnet.

### What is the specific issue or error you're encountering?

The specific issue is that the README file of the zksync-developers GitHub repository does not include an announcement regarding the deprecation of Goerli testnet and the introduction of the zkSync Sepolia Testnet

### Can you share the error messages or logs you're receiving, if any?

N/A

### Have you made any recent changes to the contract before encountering this issue?

N/A

### Are there any external libraries or contracts that your contract interacts with?

N/A

### Can you provide the relevant portions of your contract code where the issue is occurring?

N/A

### Have you tried to isolate the problem, and if so, what were the results?

N/A

### What steps have you already taken to try to resolve the issue?

I am initiating a discussion to bring attention to the missing announcement in the README file of the zksync-developers GitHub repository.

### Additional Details

The announcement regarding the deprecation of Goerli testnet and the introduction of the zkSync Sepolia Testnet is crucial for developers relying on zkSync. It provides important information about the migration process and updates to RPC endpoints, block explorers, faucets, and bridges. It's essential to ensure that this information is easily accessible to developers to facilitate a smooth transition to the new testnet.

**Comment by dutterbutter** [comment, marked as answer]
Announcement is pinned to dev discussions here: https://github.com/zkSync-Community-Hub/zksync-developers/discussions/228

It would most definitely not be in the README but rather under the announcements category in dev discussions.

**Reply by samui68400** [reply]
Thanks for your response. I initially had trouble finding the information, which prompted me to make the suggestion. Additionally, I've noticed that some links in the README are leading to 404 errors. It would be great if we could address this issue for better usability.
